tests/cases/executor_7/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# This file is automatically generated by Jina executor normalizer plugin.
```

Comment:
Can use a simple Dockerfile to verify.

Reply:
Let's use simpler one, at least remove the comments. 

and need to add one more for testing multi-stage case:

```
FROM xxx

RUN xxx

FROM xxx

RUN xxx
```